perceiverio/cli.py

```diff
@@ -9,13 +9,11 @@
     LightningArgumentParser,
 )
 
-from cli.trainer import DDPStaticGraphPlugin
-
 
```

Comment:
`from pytorch_lightning.plugins import DDPPlugin` is missing here

Reply:
yep, thx